catalog/rest/catalog-rest-impl/src/test/java/org/codice/ddf/rest/impl/action/MetacardTransformerActionProviderTest.java

```diff
@@ -108,7 +108,9 @@ public void getMetacardActionUrlWhenUrlIsMalformed() throws Exception {
     String invalidHost = "23^&*#";
     System.setProperty(SystemBaseUrl.EXTERNAL_HOST, invalidHost);
 
-    actionProvider.getMetacardActionUrl(REMOTE_SOURCE_ID, metacard);
+    MetacardTransformerActionProvider testProvider =
```

Comment:
✏️ 	Any reason we can't assign to the original `actionProvider` here? It might also be useful to add a comment here to explain why it is now necessary to reinstantiate the action provider (i.e. to pick up the system base url change)

Reply:
Added a comment to reflect that the SystemBaseUrl.EXTERNAL_HOST property lookup is cached